### PR TITLE
feat: add WA hotline URL in config and update all links that use it

### LINKED ISSUE #329: Donasi Warga page + widget
## UI Design

image

image

## Implementation Model

To make this implementation quick, let's implement the widget with dummy data from JS first, before converting said dummy data into the final Netlify CMS collection.

## Tasks

- [ ] Create Donasi Warga widget on homepage with pulling dummy data from JS
- [ ] Create Donasi Warga page using said dummy data
- [ ] Convert JS dummy data into Netlify CMS collection
- [ ] Update page + widget to pull data from new Netlify CMS collection

### LINKED ISSUE #510: Update the WhatsApp link to indicate that it's coming from our website
@resir014 in the last meeting we want to explore possibilities to include predefined message template when user is clicking the hotline button. 

So that we can analyze how many of the user that get in touch with us, is actually coming from the WBW website. 

So to enable that we need to modify the link used to include the template text. 

Would you mind to change all the link that directing user to Whatsapp to this link instead? 

https://api.whatsapp.com/send?phone=6281257579812&text=Halo%20hotline%20Warga%20Bantu%20Warga,%20saya%20saat%20ini%20membutuhkan%20bantuan%20terkait%20dengan%20covid-19

Cc: @mazipan @zainfathoni

_Originally posted by @baraeb92 in https://github.com/kawalcovid19/wargabantuwarga.com/issues/506#issuecomment-889610644_

### PR DESCRIPTION
**Pull request by resir014**
Closes #510

## Description

Fixes the WhatsApp hotline URL in the navigation bar, as well as adding it to the Netlify CMS site config section so content writers can edit it.

### CODE REVIEW

**Review by mazipan — APPROVED** [review]
Solid

File: components/layout/__tests__/navigation.test.tsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ describe("Navigation", () => {
             >
               <a
                 class="inline-flex flex-col items-center justify-center text-center h-12 px-2 rounded-md text-gray-600 hover:text-blue-600"
-                href="https://wa.me/6281257579812"
+                href="https://bit.ly/hotlinewarga"
```
Comment (zainfathoni) [inline components/layout/__tests__/navigation.test.tsx:55]: This test will be failing once the editor update the link through Netlify CMS. Please supply a hardcoded variable instead for the test. If it's not supplied through props, you can mock the value using `jest.mock` utility.

**Review by baraeb92 — APPROVED** [review]

**Review by zainfathoni — APPROVED** [review]
Approved with a minor change needed to make it resilient for Netlify CMS edits in the future.

File: components/layout/__tests__/navigation.test.tsx (hunk 1 above)
Comment (zainfathoni) [reply components/layout/__tests__/navigation.test.tsx:57]: The link is mocked in ab2d255 and the links are asserted in 00f585e

**Pull request merged**